Makefile

```
@@ -206,3 +206,7 @@ public/api/_analytics-events.json: .yardoc .yardoc/objects/root.dat
 		--no-output \
 		--db $@ \
 		-- $<
+
+update: ## Run after a git pull perhaps
```

```suggestion
update: ## Update dependencies, useful after a git pull
```
Style nit, I'd drop the "perhaps"